src/etl/gene_disease_ortho_etl.py

```diff
@@ -23,9 +23,9 @@ def __init__(self, config):
                   (pub:Publication {primaryKey:"MGI:6194238"}),
                   (ecode:EvidenceCode {primaryKey:"IEA"})
 
-                CREATE (dga:Association:DiseaseEntityJoin {primaryKey:row.uuid})
-                    SET dga.dataProvider = 'Alliance'
-                    SET dga.sortOrder = 10
+                MERGE (dga:Association:DiseaseEntityJoin {primaryKey:row.diseaseUniqueKey})
+                    ON CREATE SET dga.dataProvider = 'Alliance'
+                    ON CREATE SET dga.sortOrder = 10
 
```

Comment:
This can be done as a single:
ON CREATE SET dga.dataProvider = 'Alliance', dga.sortOrder = 10